bindata/oauth-apiserver/ds.yaml

```diff
@@ -57,7 +57,7 @@ spec:
               --etcd-keyfile=/var/run/secrets/etcd-client/tls.key \
               --etcd-certfile=/var/run/secrets/etcd-client/tls.crt \
               --shutdown-delay-duration=3s \
-              --v=2
+              ${FLAGS}
```

Comment:
let's call verbosity a standard replacer. YOu can imagine having those.  it doesn't preclude a ${FLAGS} too